packages/react/src/index.ts

```diff
@@ -1 +1 @@
-export * from "./ReactPlainBottomSheet";
+export * from "./BottomSheet";
```

Comment:
Re-export not all exactly please. Public apis should be exported carefully

Reply:
Thanks! Made an issue for this:
https://github.com/orgs/plainsheet/projects/2/views/1?pane=issue&itemId=75153379